src/windows/index.js

```diff
@@ -6,6 +7,22 @@ import { v4 as uuidv4 } from 'uuid'
 const WindowContext = React.createContext(null)
 const ClearContext = React.createContext(null)
 
+function Overlay() {
```

Comment:
There is an Overlay already inside Window. The task is to investigate why the original one doesn't block input.